tests/test_cargo_test.rs

```diff
@@ -46,6 +46,41 @@ test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured
         RUNNING)));
 });
 
+test!(cargo_test_release {
+    let p = project("foo")
+        .file("Cargo.toml", basic_bin_manifest("foo").as_slice())
```

Comment:
`as_slice()` is deprecated (and failing your travis build). you'll want to rebase and change this to `&basic_bin_manifest("foo")`